examples/pennylane_skip_pytorch_noise.py

```diff
@@ -0,0 +1,205 @@
+#
+# Now that we have initialized the device, we can construct our quantum node. Like the
+# other tutorials, we use the :mod:`qnode decorator <pennylane.decorator>` to convert
+# our quantum function (encoded by the circuit above) into a quantum node
+# running on the QVM.
+
+
+@qml.qnode(dev, interface="torch")
+def circuit(phi, theta):
+    qml.RX(theta, wires=0)
+    qml.RZ(phi, wires=0)
+    return qml.expval(qml.PauliZ(0))
+
+
+##############################################################################
+# To make the QNode 'PyTorch aware', we need to specify that the QNode interfaces
+# with PyTorch. This is done by passing the ``interface='torch'`` keyword argument.
+#
+# As a result, this QNode will be set up to accept and return PyTorch tensors, and will
+# also automatically calculate any analytic gradients when PyTorch performs backpropagation.
+
+
+##############################################################################
+# Optimization
+# ------------
+#
+# We can now create our optimization cost function. To introduce some additional
+# complexity into the system, rather than simply training the variational circuit
+# to 'flip a qubit' from state :math:`\ket{0}` to state :math:`\ket{1}`, let's also
+# modify the target state every 100 steps. For example, for the first 100 steps,
+# the target state will be :math:`\ket{1}`; this will then change to :math:`\ket{0}`
+# for steps 100 and 200, before changing back to state :math:`\ket{1}` for steps 200
+# to 300, and so on.
+
+
+def cost(phi, theta, step):
+    target = -(-1) ** (step // 100)
+    return torch.abs(circuit(phi, theta) - target) ** 2
+
+
+##############################################################################
+# Now that the cost function is defined, we can begin the PyTorch optimization.
+# We create two variables, representing the two free parameters of the variational
+# circuit, and initialize an Adam optimizer:
+
+phi = Variable(torch.tensor(1.0), requires_grad=True)
+theta = Variable(torch.tensor(0.05), requires_grad=True)
+opt = torch.optim.Adam([phi, theta], lr=0.1)
+
+##############################################################################
+# As we are using the PyTorch interface, we must use PyTorch optimizers,
+# *not* the built-in optimizers provided by PennyLane. The built-in optimizers
+# only apply to the default NumPy/autograd interface.
+#
+# Optimizing the system for 400 steps:
+
+for i in range(400):
+    opt.zero_grad()
+    loss = cost(phi, theta, i)
+    loss.backward()
+    opt.step()
+
+##############################################################################
+# We can now check the final values of the parameters, as well as the final
+# circuit output and cost function:
+
+print(phi)
+print(theta)
+print(circuit(phi, theta))
+print(cost(phi, theta, 400))
+
+##############################################################################
+# .. rst-class:: sphx-glr-script-out
+#
+#  Out:
+#
+#  .. code-block:: none
```

Comment:
These look so terrible in the rendered notebooks :frowning:

Reply:
:/ I'll see what I can do

Reply:
Should we have a long term aim to have _all_ of the examples executed, to avoid faking the output? This would require migrating away from readthedocs, and having our own home-spun cloud service to build and deploy the docs

Reply:
What about the ones that take a long time to execute?

Reply:
I've updated as many to run as I can. They take the following time to execute:
```console
computation time summary:
    - ../examples/pennylane_run_variational_classifier.py: 1.8e+02 sec
    - ../examples/pennylane_run_state_preparation.py: 97 sec
    - ../examples/pennylane_run_QGAN.py: 28 sec
    - ../examples/pennylane_run_plugins_hybrid.py: 11 sec
    - ../examples/pennylane_run_variational_quantum_eigensolver.py: 3.1 sec
    - ../examples/pennylane_run_qubit_rotation.py: 0.33 sec
    - ../examples/pennylane_run_gaussian_transformation.py: 0.14 sec
    - ../examples/pennylane_run_advanced_usage.py: 0.077 sec
    - ../examples/pennylane_quantum_neural_net.py: 0 sec
    - ../examples/pennylane_pytorch_noise.py: 0 sec
```

Of the two that are being skipped, the QNN takes too long to execute, and the PyTorch one uses a QPU.

Reply:
this is pretty good coverage, probably sufficient to keep just those two out of autorunning